.gitignore

```diff
@@ -1,2 +1,4 @@
+.gitignore
```

Comment:
we should not ignore the gitignore itself, right?

Reply:
sorry I did not mean that to get in, I meant to remove the change. It was caused because I ran phpunit from within my local Coder git repository and it created that cache file, which kept getting in the way so I aded it to .gitignore. Then that introduced the change to .gitigore itself. I always assumed that each projects .gitignore is ignored.
  
Actually I do have .gitignore in my global ignore file. I'm not sure about whether its right to ignore or not ignore it per project. I always thought it was added for the convenience of developers, and it is not really part of the project.